src/ui/interface.js

```diff
@@ -68,6 +68,21 @@ module.exports = function Interface(options) {
       return true;
     });
 
+    $("#sample-image__select").click(function(e){
+      e.stopPropagation();
+      const img = (e.target.classList.contains('rfi')) ? document.getElementById('rfi') : document.getElementById('bfi');
```

Comment:
Here's another super great line of code - i love that the idea is that you can add the "bfi" or "rfi" classes to switch between. Maybe we should explain this too in a comment!

Reply:
Aha - i see now -- we want to click on the thumbnail, but use the big hidden image! I wonder - if we use relatively low res sample images, like, 1024x768 or less, could we simplify things by just skipping thumbnails? JPGs don't weigh that much, you know? Then the logic here can be much simpler - we just display the big images with `style="width:200px;"` or something like that!

Reply:
So on this line, if we stop using separate thumbnails, we can maybe just skip using red-filter or blue-filter classes completely. Since we'd just select whatever image, and think about what kind of filter later?